apps/client/src/features/rundown/event-block/composite/EventBlockChip.tsx

```diff
@@ -75,3 +78,51 @@ function EventUntil(props: EventUntilProps) {
     </Tooltip>
   );
 }
+
+interface EventReportProps {
+  className: string;
+  id: string;
+  duration: number;
+}
+
+function EventReport(props: EventReportProps) {
+  const { className, id, duration } = props;
+  const { data } = useReport();
+  const currentReport: OntimeEventReport | undefined = data[id];
+
+  const [value, isOver] = useMemo(() => {
+    if (!currentReport) {
+      return [null, false];
+    }
+
+    const { startedAt, endedAt } = currentReport;
+    if (!startedAt || !endedAt) {
+      return [null, false];
+    }
+
+    const actualDuration = endedAt - startedAt;
+    const difference = actualDuration - duration;
+    const absDifference = Math.abs(difference);
+
+    if (absDifference < MILLIS_PER_SECOND) {
+      return ['==', false]; //TODO: how to symbolize ontime?
+    }
+
+    const isOver = difference > 0;
+
+    const value = `${isOver ? '+' : '-'}${formatDuration(Math.abs(difference), difference > 2 * MILLIS_PER_MINUTE)}`;
+    return [value, isOver];
+  }, [currentReport, duration]);
+
+  if (!value) {
+    return null;
+  }
+
+  return (
+    <Tooltip label='Difference from the expected duration' openDelay={tooltipDelayFast}>
+      <div className={cx([style.chip, isOver ? style.over : style.under, className])}>
+        <div>{value}</div>
```

Comment:
suggestion, we dont need this extra div
```suggestion
        {value}
```